firmware/Core/Inc/telecommands/eps_telecommands.h

```diff
@@ -91,6 +91,10 @@ uint8_t TCMDEXEC_eps_get_piu_housekeeping_data_run_avg_json(
     char *response_output_buf, uint16_t response_output_buf_len
 );
 
+uint8_t TCMDEXEC_eps_get_list_of_enabled_channels(
```

Comment:
Please rename to `TCMDEXEC_eps_get_enabled_channels_json` (ie remove "list of"). Will give the result as JSON.